src/azure-cli/azure/cli/command_modules/appservice/_params.py

```diff
@@ -67,6 +67,11 @@ def load_arguments(self, _):
         help='The Isolated pricing tiers, e.g., I1 (Isolated Small), I2 (Isolated Medium), I3 (Isolated Large)',
         arg_type=get_enum_type(['I1', 'I2', 'I3']))
 
+    static_web_app_sku_arg_type = CLIArgumentType(
+        help='The pricing tiers for Static Web Spp, e.g., F1(Free), S1(Standard)',
```

Comment:
so I think the sku's are just Free and Standard, we don't have F1 or S1

Reply:
removed